azure-proto-core/Resources/IdentityKind.cs

```diff
@@ -30,12 +30,17 @@ public int CompareTo(string other)
 
         public bool Equals(IdentityKind other)
         {
-            throw new NotImplementedException();
+            if (Object.ReferenceEquals(null, other))
```

Comment:
Any tests for updates in IdentityKind class?

Reply:
IdentityKind.cs hasn't been completed - it will have it's own tests in a separate PR. I only filled two methods so I could get Identity.cs to work.